docs/API.md

```diff
@@ -451,77 +499,146 @@ New version:
+which is not the same -- they differ in size, and mistaking them may end up
+with a crash. This must be kept in mind especially in any C wrapper. For
+convenience, the setting option function may accept both `int32_t` and `bool`
+types, but this is not so in the case of getting an option value.
+
+**UDT project legacy note**: Almost all options from the UDT library are
+derived (there are a few deleted, including some deprecated already in UDT),
+many new SRT options have been added. All options are available exclusively
+with the `SRTO_` prefix. Old names are provided as alias names in the `udt.h`
+legacy/C++ API file. Note the translation rules:
 * `UDT_` prefix from UDT options was changed to the prefix `SRTO_`
 * `UDP_` prefix from UDT options was changed to the prefix `SRTO_UDP_`
 * `SRT_` prefix in older SRT versions was changed to `SRTO_`
 
-The Binding column should define for these options one of the following
-statements concerning setting a value:
+The below table shows the characteristics of the options. Legend:
+
+1. **Since**
+
+Defines the SRT version when this option was first introduced. If this field
+is empty, it's an option derived from UDT. Version 0.0.0 is the oldest version
+of SRT ever created and put into use.
+
+2. **Binding**
+
+Defines limitation on setting the option (the field is empty if the option
+is not settable, see **Dir** column):
+
+* pre: For a connecting socket (both as caller and rendezvous) it must be set
+prior to calling `srt_connect()` or `srt_bind()` and never changed thereafter.
+For a listener socket it should be set to a listening socket and it will be
+derived by every socket returned by `srt_accept()`.
 
-* pre: For connecting a socket it must be set prior to calling `srt_connect()`
-and never changed thereafter. For a listener socket it should be set to a binding
-socket and it will be derived by every socket returned by `srt_accept()`.
 * post: This flag can be changed any time, including after the socket is
-connected. On binding a socket setting this flag is effective only on this
-socket itself. Note though that there are some post-bound options that have
-important meaning when set prior to connecting.
+connected (as well as on an accepted socket). On a listening socket setting this
+flag is effective only on this socket itself. Note though that there are some
+post-bound options that have important meaning when set prior to connecting.
+
+3. **Type**
+
+The data type of the option; see above.
+
+4. Units
+
+Roughly specified unit, if the value defines things like length or time.
+It can also define more precisely, what kind of specialization can be used
+when the type is integer:
+
+* enum: the possible values are defined in an enumeration type
+* flags: the integer value is a collection of bit flags
+
+5. Default
+
+The exact default value, if it can be easily specified. For more complicated
+state of the default state of particular option, it will be explained in the
```

Comment:
```suggestion
The exact default value, if it can be easily specified. For a more complicated
default state of particular option, it will be explained in the
```